src/components/Condition.tsx

```diff
@@ -0,0 +1,20 @@
+interface ConditionProp {
+  condition: string;
+}
+
+const Condition = ({ condition }: ConditionProp) => {
+  const getCondition = () => {
+    switch (condition) {
+      case 'A':
+        return 'Salvage Title';
+      case 'N':
+        return 'New';
+      default:
+        return 'N/A';
+    }
+  };
+
+  return <div>{getCondition()}</div>;
+};
+
+export default Condition;
```

Comment:
I'm not sure this warrants a component, just put the function somewhere and use it where you need to render a condition string

Reply:
I fixed it